src/Aliencube.AzureFunctions.Extensions.OpenApi.Core/DocumentHelper.cs

```diff
@@ -40,7 +40,7 @@ public DocumentHelper(RouteConstraintFilter filter, IOpenApiSchemaAcceptor accep
         }
 
         /// <inheritdoc />
-        public List<MethodInfo> GetHttpTriggerMethods(Assembly assembly)
+        public IList<MethodInfo> GetHttpTriggerMethods(Assembly assembly)
```

Comment:
I wouldn't buy this &ndash; `IList<T>` and `IDictionary<T1, T2>`.

Reply:
Ok - reverted